GameServerLib/Logic/Content/ItemCollection.cs

```diff
@@ -6,57 +6,12 @@
 
 namespace LeagueSandbox.GameServer.Logic.Content
 {
-    public class ContentCollectionEntry
+    public class ContentCollectionEntry : ContentFile
```

Comment:
It has to much references and i'm afraid it will conflict with Gan's commit so i'll just leave it like that.
Nothing that can't get easily renamed latter ...